exporter/collector/metrics.go

```diff
@@ -434,7 +434,7 @@ func (me *MetricsExporter) exportToTimeSeries(ctx context.Context, req *monitori
 func (me *MetricsExporter) export(ctx context.Context, req *monitoringpb.CreateTimeSeriesRequest) error {
 	// if this is an empty request, skip it
 	// empty requests are used by the WAL to signal the end of pending data
-	if len(req.String()) == 0 {
+	if req == nil || (req.TimeSeries == nil && req.Name == "") {
```

Comment:
Idea for readability: define a helper function `isReqEmpty` -- compiler will likely inline it anyway.